packages/affiliates/libs/start.js

```diff
@@ -0,0 +1,19 @@
+// This file is a common bootstrapping library to initialize configs and web3 apps in the /apps dir.
+// It expects a function to be passed in. This file is repsonsible for constructing the configuration
+// file and other dependencies for that function and injecting it. The idea is that any app would share
+// the same pattern for getting configuration and dependencies.
+// example: see how its used in apps/DeployerRewards
+require("dotenv").config();
+const assert = require("assert");
+const Web3 = require("web3");
+const Path = require("path");
+const [, , configPath] = process.argv;
+
+assert(configPath, "Requires relative path to config params");
+const config = require(Path.join(process.cwd(), configPath));
+assert(process.env.CUSTOM_NODE_URL, "requires CUSTOM_NODE_URL");
```

Comment:
nit: should we perform this assert logic inside the function so that this file won't error out if transitively included by something that doesn't end up calling this start function?

Reply:
Agreed, since `web3` is in the App's constructor this makes sense